R/loadFromQIIME2.R

```diff
@@ -154,7 +146,8 @@ loadFromQIIME2 <- function(featureTableFile,
     } else {
         refseq <- NULL
     }
-
+    
+    feature_tab <- .set_feature_tab_dimnames(feature_tab, sample_meta, taxa_tab)
```

Comment:
This line could be added to other import functions also, especially to `loadFromMothur` because there is a risk of name mismatch

Reply:
It is possible to open another PR for the other import functions but also fine to add to this PR if this can be easily done.

Reply:
OK, I can do that, this is a fork so I think it's not that straight-forward, but I can do that after merging this